lib/config.js

```diff
@@ -9,6 +9,7 @@ var autoconfig = function(overrides) {
     dataTopicPrefix: ':cloud:data',
     persistentStore: process.env.WFM_USE_MEMORY_STORE !== "true",
     syncOptions: {
+      "syncFrequency": 5,
```

Comment:
This looks like joke :) How users will know sync_frequency is related to client and sync_frequency to server. Oh.. wait..